pkg/template/api/v1/types.go

```diff
@@ -14,15 +14,23 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
-	// Objects is an array of objects to include in this template. Required.
+	// message is an optional instructional message that will
+	// be displayed when this template is instantiated.
```

Comment:
What do you mean by instantiated?  You can't output this message in the `oc process template | oc create -f` flow, right?  We're adding a template API field for the the `new-app` only flow?  @jwforres would need to special case this in the webconsole too.